pk_tic_tac_toe.rb

```diff
@@ -1,17 +1,28 @@
 class TicTacToeGame
+  VICTORY = [
+    [0, 1, 2], # same gp top row.
+    [3, 4, 5], # same gp middle row.
+    [6, 7, 8], # same gp bottom row.
+    [0, 3, 6], # same gp left column.
+    [1, 4, 7], # same gp center column.
+    [2, 5, 8], # same gp right column.
+    [0, 4, 8], # same gp left diagonal.
+    [6, 4, 2]  # same gp right diagnal.
+  ]
+
   attr_accessor :board
 
   def initialize
     @board = [*1..9]
-    @@remaining_spaces = 9
+    @remaining_spaces = 9
   end
 
-  def self.remove_space
-    @@remaining_spaces -= 1
+  def remove_space
```

Comment:
After changing the "remaining_spaces" class variable to an instance variable, there was no need to prepend this class method with the "self" keyword; I changed this class method from "self.remove_space" to "remove_space"